applications/StructuralMechanicsApplication/custom_strategies/custom_convergencecriterias/displacement_and_other_dof_criteria.h

```diff
@@ -420,7 +420,7 @@ class DisplacementAndOtherDoFCriteria : public ConvergenceCriteria< TSparseSpace
                 }
                 else
                 {
-                    mReferenceOtherDoFNorm += AuxValue*AuxValue;
+                    DeltaOtherDoFNorm += AuxValue*AuxValue;
```

Comment:
Uff, that it is my fault

Reply:
no worries
I was always curious why it always said that the rotations converge instant, but never checked in detail why